rdmo/options/providers.py

```diff
@@ -9,13 +9,13 @@ class Provider(Plugin):
     # determines if the page needs to be refreshed after a value is stored
     refresh = False
 
-    def get_options(self, project, search=None):
+    def get_options(self, project, search=None, user=None, site=None):
         raise NotImplementedError
 
 
 class SimpleProvider(Provider):
```

Comment:
I don't understand? You mean the whole function? This is the usual pattern if you expect an inherited class to implement the function in python. If you don't, you get NotImplementedError instead of a more generic Exception.

Reply:
no I mean the `SimpleProvider` class, it is not used anywhere except for in the tests right? ;)

Reply:
Yes, but also as a template to show how the plugin is implemented. I would like to keep it here.